tsconfig.json

```diff
@@ -8,6 +8,7 @@
     "lib": ["es6", "dom"],
     "sourceMap": true,
     "allowJs": true,
+    "allowSyntheticDefaultImports": true,
```

Comment:
why is this necessary?

Reply:
ApolloConsumer default imports the React library